Makefile

```
@@ -1,15 +1,30 @@
-.PHONY: build test generate clean lint release-install-signing release-clean-signing release-dmg
+.PHONY: build test test-stable test-all generate clean lint release-install-signing release-clean-signing release-dmg
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)